src/Service/MigrationService.php

```diff
@@ -450,7 +414,8 @@ private function createMigrationHistoryTable(): void
         $this->consoleHelper->io()->section("Creating migration history table \"$tableName\"...");
 
         $this->db->createCommand()->createTable($this->migrationTable, [
-            'version' => 'varchar(' . $this->maxNameLength . ') NOT NULL PRIMARY KEY',
+            'id' => 'pk',
```

Comment:
Why?

Reply:
For example, we have several migration and run migration. Migration fast and applied by one second: `apply_time` equal. Migration history can return migrations with incorrect orderd. `id` resolve this problem.